test/functional/p2p_syncing_test.py

```diff
@@ -13,7 +13,7 @@ class ExampleTest(BitcoinTestFramework):
     def set_test_params(self):
         self.setup_clean_chain = True
         self.num_nodes = 2
-        self.extra_args = [[], []]
+        self.extra_args = [["--max-tip-age=63070000"], ["--max-tip-age=63070000"]]
```

Comment:
Why does this have to be manually put here instead of using the default value? This doesn't look right.

Reply:
This is needed because w use the pre-made blocks (by the `generate_blocks_for_functional_tests` test), so all timestamps are old and the initial block download is never finished with the default max tip age value.

Reply:
Please put a TODO on this to remove it later.